src/tv2_afvd_showstyle/helpers/pieces/__tests__/grafikViz.spec.ts

```diff
@@ -123,7 +123,8 @@ describe('grafik piece', () => {
 					type: Tv2PieceType.GRAPHICS,
 					outputLayer: Tv2OutputLayer.OVERLAY,
 					partType: PartType.Kam,
-					pieceExternalId: dummyPart.externalId
+					pieceExternalId: dummyPart.externalId,
+					graphicsTemplateName: 'bund'
```

Comment:
Can we get that string from some constant, enum or object as `"dummyPart.externalId"`

Reply:
It isn't set up that way at the moment it seems, so for this PR i won't get that far into changing tests. But a very good point for the future!